.github/workflows/ruff.yaml

```diff
@@ -296,3 +352,10 @@ jobs:
       - name: "Update pre-commit mirror"
         run: |
           curl -X POST -H "Accept: application/vnd.github+json" -H "Authorization: Bearer ${{ secrets.RUFF_PRE_COMMIT_PAT }}" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/repos/charliermarsh/ruff-pre-commit/dispatches --data '{"event_type": "pypi_release"}'
+      - uses: actions/download-artifact@v3
+        with:
+          name: bin
+      - name: "Upload binaries to release"
+        run: |
+          RELEASE_TAG=$(git describe --tags --abbrev=0)
```

Comment:
@charliermarsh since the action itself isn't creating the release (ie. after a tag is pushed by some other automation), this seemed like the easiest way to get the latest release name

Reply:
Can we somehow get this from `github.ref`, similar to how the `if: "startsWith(github.ref, 'refs/tags/')"` operation works above?